scripts/next-data/generateNodeReleasesData.mjs

```diff
@@ -0,0 +1,41 @@
+import nodevu from '@nodevu/core';
+import { writeFile } from 'fs/promises';
+import { join } from 'path';
```

Comment:
why not?
```suggestion
import { join } from 'node:path';
```

Reply:
Just checked other imports, I will add it for consistency, thanks!